fission/src/mirabuf/MirabufInstance.ts

```diff
@@ -149,8 +149,10 @@ class MirabufInstance {
                         opacity: opacity,
                         transparent: opacity < 1.0,
```

Comment:
Use the `shadowSide` parameters to render against both the front and back faces when casting the shadow. This will remove the gap from the object to the shadow caused by the normal bias.